grafana-plugin/src/types.ts

```diff
@@ -28,13 +28,6 @@ export enum OnCallPluginExtensionPoints {
 declare global {
   export interface Window {
     // https://github.com/grafana/grafana/blob/78bef7a26a799209b5307d6bde8e25fcb4fbde7d/public/views/index-template.html#L251-L258
-    grafanaBootData: {
```

Comment:
This seems to be already typed by grafana after package bump, therefore not needed anymore